src/hooks.js

```diff
@@ -56,18 +56,22 @@ export function useArrows({ onUp, onLeft, onDown, onRight } = {}) {
 
 function stepsReducer(state, { type, value, steps }) {
   const { step } = state
+  const stepsCount = steps - 1
 
   let newStep = null
 
   if (type === 'set') {
     newStep = value
   }
-  if (type === 'next' && step < steps - 1) {
+  if (type === 'next' && step < stepsCount) {
```

Comment:
More of a stylistic thing (and I'm honestly not sure what is common outside in the wilder React community or documentation), but should we use `if else`s for these type matches?

Reply:
If anything I'd say switch statements are most common, but I don't have strong opinions on it. I think this works fine.